scripts/flush_db.py

```diff
@@ -53,9 +57,28 @@ async def main():
 
     await batch_insert(events)
 
-    # await redis.flushall()
+    await redis.flushall()
+
+
+async def redis_count():
+    redis = await Redis.from_url(str(REDIS_URL))
+    return await redis.dbsize()
 
 
 if __name__ == "__main__":
-    loop = asyncio.get_event_loop()
-    loop.run_until_complete(main())
+    import sys
+
+    if len(sys.argv) != 2:
```

Comment:
is there a reason why we need arguments to this script? I'm not sure we only want to flush 10K items if there isn't a `full` argument, I think we want to flush all of them regardless